molgenis-python/src/main/resources/python/molgenis.py

```diff
@@ -0,0 +1,72 @@
+import requests;
+import json;
+import urllib;
+
+class Connection():
+	def __init__(self, url):
+	    self.url = url
+
+	def login(self, username, password):
+		response = requests.post(self.url + "v1/login", 
```

Comment:
Would it be possible to use requests.Session()? This helps a lot in speed when adding many rows